src/AppInstallerCLICore/Workflows/WorkflowBase.cpp

```diff
@@ -851,6 +851,7 @@ namespace AppInstaller::CLI::Workflow
         auto& searchResult = context.Get<Execution::Data::SearchResult>();
 
         Logging::Telemetry().LogSearchResultCount(searchResult.Matches.size());
+        AICLI_LOG(Repo, Verbose, << "Search result size: " << searchResult.Matches.size());

```

Comment:
```suggestion
        AICLI_LOG(Cli, Verbose, << "Search result size: " << searchResult.Matches.size());

```

Also, I think we should move this inside LogSearchResultCount() above to be consistent with other telemetry logging too?